runtests.sh

```diff
@@ -101,6 +115,22 @@ function compile_cpp {
     fi
 }
 
+function clang_format {
+    echo "Running clang-format..."
+    ${cmdPrefix}python -m tests.clang_format_utils
+    clang_format_tool='.clang-format-bin/clang-format'
```

Comment:
If we use the pip clang-format we would check it's present here and use just `clang-format` as the tool name.